apps/web/src/services/alert-manager-v2/components/ServiceCreateStep2.vue

```diff
@@ -0,0 +1,130 @@
+<script setup lang="ts">
+import { computed, onMounted, reactive } from 'vue';
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+import { PSelectCard, PLazyImg } from '@cloudforet/mirinae';
+
+import type { ListResponse } from '@/schema/_common/api-verbs/list';
+import type { PluginGetVersionsParameters } from '@/schema/repository/plugin/api-verbs/get-versions';
+import type { PluginListParameters } from '@/schema/repository/plugin/api-verbs/list';
+import type { PluginModel } from '@/schema/repository/plugin/model';
+import type { RepositoryListParameters } from '@/schema/repository/repository/api-verbs/list';
+import type { RepositoryModel } from '@/schema/repository/repository/model';
+import { i18n } from '@/translations';
+
+import { assetUrlConverter } from '@/lib/helper/asset-helper';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import ServiceCreateStepContainer from '@/services/alert-manager-v2/components/ServiceCreateStepContainer.vue';
+import { useServiceFormStore } from '@/services/alert-manager-v2/store/service-form-store';
+
+const serviceFormStore = useServiceFormStore();
+const serviceFormState = serviceFormStore.state;
+
+const storeState = reactive({
+    currentSubStep: computed<number>(() => serviceFormState.currentSubStep),
+});
+const state = reactive({
+    versionLoading: false,
+    webhookTypeList: [] as PluginModel[],
+    isAllFormValid: false,
+    selectedWebhookType: {} as PluginModel,
+    pluginVersions: undefined as undefined|string,
+});
+
+const handleSelectWebhook = () => {
+    state.isAllFormValid = true;
+    serviceFormStore.setSelectedWebhookTypeId(state.selectedWebhookType.plugin_id);
+    getVersions();
+};
+
+const getRepositoryID = async () => {
+    const res = await SpaceConnector.clientV2.repository.repository.list<RepositoryListParameters, ListResponse<RepositoryModel>>({
+        repository_type: 'remote',
+    });
+    return res.results ? res.results[0].repository_id : '';
+};
```

Comment:
When returning something from a function, it would be good if the type was specified.